ontotext-yasgui-web-component/src/models/yasgui-configuration.ts

```diff
@@ -190,8 +189,7 @@ export const defaultYasqeConfig: Record<string, any> = {
     {name: 'createSavedQuery', visible: true},
     {name: 'showSavedQueries', visible: true},
     {name: 'shareQuery', visible: true},
-    {name: 'includeInferredStatements', visible: true},
-    {name: 'expandResultsOverSameAs', visible: true}
```

Comment:
So, you removed the includeInferredStatements but left expandResultsOverSameAs. Why is that?